components/brave_wallet_ui/components/buy-send-swap/swap/index.tsx

```diff
@@ -75,6 +78,30 @@ function Swap (props: Props) {
     onFilterAssetList(toAsset)
   }
 
+  const submitText = React.useMemo(() => {
+    if (validationError === 'insufficientBalance') {
+      return 'Insufficient balance'
```

Comment:
I think these should all be localized. 😎

Reply:
Done!